src/dispatch/incident/scheduled.py

```diff
@@ -32,6 +43,47 @@
 log = logging.getLogger(__name__)
 
 
+@scheduler.add(every(1).hours, name="incident-tagger")
+@background_task
+def auto_tagger(db_session):
+    """Attempts to take existing tags and associate them with incidents."""
+    tags = tag_service.get_all(db_session=db_session).all()
+    log.debug(f"Fetched {len(tags)} tags from database.")
+
+    tag_strings = [t.name.lower() for t in tags if t.discoverable]
+    phrases = build_term_vocab(tag_strings)
+    matcher = build_phrase_matcher("dispatch-tag", phrases)
+
+    p = plugins.get(
+        INCIDENT_PLUGIN_STORAGE_SLUG
+    )  # this may need to be refactored if we support multiple document types
```

Comment:
Just adding a note that we will need to refactor this once https://github.com/Netflix/dispatch/pull/223 lands.

Reply:
Yes eventually, but #223  itself won't require a refactor of this code. We will move each plugin type one by one and refactor all the slug references.